### PR TITLE
Prevent debug on client 11 after first mob kill

### PR DESCRIPTION
**Pull request by Oen44**

### CODE REVIEW

File: data/modules/scripts/bestiary/bestiary.lua [hunk 1]
```diff
@@ -554,7 +553,9 @@ function Player.addBestiaryKill(self, monsterID) --MonsterID can be Name
 	local curCount = self:getBestiaryKillCount(monsterID)
 	local monster = Bestiary.Monsters[monsterID]
 	if curCount == 0 then
-		self:sendBestiaryEntryChanged(monsterID)
+		if self:getClient().version == 1200 then
```
Comment (gpedro) [inline data/modules/scripts/bestiary/bestiary.lua]: should be `>= 1200`, uhm?

Comment (Oen44) [reply data/modules/scripts/bestiary/bestiary.lua]: Is there client higher than 12? Qt client is 1200.

Comment (gpedro) [reply data/modules/scripts/bestiary/bestiary.lua]: we don't support +12 yet, but probably this packet is used in versions higher than 12...

Comment (Oen44) [reply data/modules/scripts/bestiary/bestiary.lua]: Updated

**Review by gpedro — APPROVED** [review]

**Pull request merged**